code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -371,11 +382,113 @@
 	has_organ = list(
 		"brain" = /obj/item/organ/brain/slime
 		)
-		
+
+	has_limbs = list(
+		"chest" =  list("path" = /obj/item/organ/external/chest/slime),
+		"groin" =  list("path" = /obj/item/organ/external/groin/slime),
+		"head" =   list("path" = /obj/item/organ/external/head/slime),
+		"l_arm" =  list("path" = /obj/item/organ/external/arm/slime),
+		"r_arm" =  list("path" = /obj/item/organ/external/arm/right/slime),
+		"l_leg" =  list("path" = /obj/item/organ/external/leg/slime),
+		"r_leg" =  list("path" = /obj/item/organ/external/leg/right/slime),
+		"l_hand" = list("path" = /obj/item/organ/external/hand/slime),
+		"r_hand" = list("path" = /obj/item/organ/external/hand/right/slime),
+		"l_foot" = list("path" = /obj/item/organ/external/foot/slime),
+		"r_foot" = list("path" = /obj/item/organ/external/foot/right/slime)
+		)
+
 	suicide_messages = list(
 		"is melting into a puddle!",
 		"is turning a dull, brown color and melting into a puddle!")
 
+/datum/species/slime/handle_life(var/mob/living/carbon/human/H)
+	// Slowly shifting to the color of the reagents
+	if(H.reagents.total_volume > 0.1)
+		var/blood_amount = H.vessel.total_volume
+		var/r_color = mix_color_from_reagents(H.reagents.reagent_list)
+		var/new_body_color = BlendRGB(r_color, rgb(H.r_skin, H.g_skin, H.b_skin), blood_amount/(blood_amount+(H.reagents.total_volume)))
+		var/list/new_color_list = ReadRGB(new_body_color)
+		H.r_skin = new_color_list[1]
+		H.g_skin = new_color_list[2]
+		H.b_skin = new_color_list[3]
+		if(world.time % 200 == 42) // Once every 20 seconds - update_body is expensive
+			for(var/organname in H.organs_by_name)
+				var/obj/item/organ/external/E = H.organs_by_name[organname]
+				if(E.dna.species == "Slime People")
+					E.sync_colour_to_human(E)
+			H.update_body()
+	return ..()
+
+/mob/living/carbon/human/proc/regrow_limbs()
+	set category = "IC"
+	set name = "Regrow Limbs"
+	set desc = "Regrow one of your missing limbs at the cost of a large amount of hunger"
+
+	var/const/hungercost = 200
+	var/const/minhunger = 400
+
+	if(stat || paralysis || stunned || weakened)
```

Comment:
Because I don't want this to be unusable when handcuffed - since the action doesn't require hands, but still requires conscious thought.